crypto/include/fdoCrypto.h

```diff
@@ -14,16 +14,14 @@
 #include "base64.h"
 #include "fdoCryptoCtx.h"
 
-#ifdef KEX_ECDH384_ENABLED
-#define SEK_KEY_SIZE 32
-#define SVK_KEY_SIZE 64
+#if defined KEX_ECDH384_ENABLED
```

Comment:
The common usage is `#if defined(KEX_ECDH384_ENABLED)`. This is used in other places in the code too, let's change all instances in this PR to that use.